doc/customize.md

```diff
@@ -377,6 +377,8 @@ You can edit notebook metadata as follows
 
 ![](../examples/metadata.png)
 
+If settings do not take effect, try refreshing the page of the web browser.
```

Comment:
This is usually happening because the browser cache is a pretty hungry monster, maybe we can add here a 2 lines instruction (or link) about hard-refreshing the browser?